src/worker/clientcontrol.js

```diff
@@ -173,9 +173,9 @@ commands.CHANGENETWORK = {
             con.writeStatus('Usage: changenetwork [network_name] option-value');
         }
 
-        // Either get the specified netwrk or default to the active network
+        // Either get the specified network or default to the active network
         let network = null;
-        if (parts[0].indexOf('=' === -1)) {
+        if (parts[0].indexOf('=') === -1) {
             network = await con.userDb.getNetworkByName(con.state.authUserId, parts[0]);
             if (!network) {
                 con.writeStatus(`Network ${netName} could not be found`);
```

Comment:
While you're here, `netName` should become `parts[0]`

Reply:
oops i did notice that and forgot to change it, one moment will force push